data/Pushpa-Mali.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Pushpa Mali",
+  "type": "personal",
+  "bio": "Hi all,My name is Pushpa Mali.I have keen interest in web development.My tech skills are HTML,CSS,Javascript,MySQL,Java",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PushpaMali23",
+      "icon": "FaTwitter"
+    },
+{
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/pushpa-mali-a842b3228/",
+      "icon": "FaLinkedIn"
```

Comment:
Hi @Pushpa-Mali, thank you for contributing  :tada:
Please replace its value with **' FaLinkedin '**.
```Suggestion
      "icon": "FaLinkedin"
```